### PR TITLE
Fixes coderskirt

### PR DESCRIPTION
**Pull request by PowerfulBacon**
Schoopid coder that made the skirt can't even code wtf.
Fixes the skirt being 1 pixel off on the right sprite, fixes the thing not recognising the top contributor and fixes the skirt not even equipping at all.

:cl:
fix: coderskirt
/:cl:

### CODE REVIEW

**Review by qwertyquerty — DISMISSED** [review]

**Comment by PowerfulBacon** [comment]
hold on im actually testing it this time, wait 10 seconds

**Comment by PowerfulBacon** [comment]
dont merge it

**Comment by PowerfulBacon** [comment]
ok there you go

**Review by qwertyquerty — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
I would use the excuse I am tired, but it is 2 pm

**Review by qwertyquerty — CHANGES_REQUESTED** [review]
wait bruh you made the sprites even worse

**Comment by PowerfulBacon** [comment]
feature

**Comment by PowerfulBacon** [comment]
@qwertyquerty Fixed offset and blinking shit

**Review by qwertyquerty — APPROVED** [review]

File: code/modules/jobs/job_types/_job.dm [hunk 1]
```diff
@@ -184,11 +184,11 @@
...
 
-	if(preference_source?.ckey == get_top_contrib())
+	if(ckey(preference_source?.ckey) == ckey(get_top_contrib()))
 		var/obj/item/clothing/under/dress/skirt/coder/coderskirt = new()
-		if(H.wear_suit)
-			H.dropItemToGround(H.wear_suit)
-		if(!H.equip_to_slot_if_possible(coderskirt, SLOT_WEAR_SUIT))
+		if(H.w_uniform)
+			H.dropItemToGround(H.w_uniform)
+		if(!H.equip_to_slot_if_possible(coderskirt, SLOT_W_UNIFORM))
```
Comment (ike709) [inline code/modules/jobs/job_types/_job.dm:191]: You didn't even update the slot define on the skirt itself you fool.

**Pull request merged**